src/stories/components/404/not-found-component-404.tsx

```diff
@@ -0,0 +1,207 @@
+import Background404Dark from '../../../../public/assets/img/background-dark-404.png';
+import BackgroundMobile404 from '../../../../public/assets/img/background-mobile-404.png';
+import BackgroundMobile404Dark from '../../../../public/assets/img/background-mobile-dark-404.png';
+import Logo404 from '../../../../public/assets/img/logo-404.png';
+import Logo404Dark from '../../../../public/assets/img/logo-dark-404.png';
+import { Typography, useMediaQuery } from '@mui/material';
+import lightTheme from '../../../../styles/theme/light';
+import { CustomButton } from '../custom-button/custom-button';
+import { useRouter } from 'next/router';
+import { useThemeContext } from '../../../core/context/ThemeContext';
+
+const CardNotFoundPage: NextPage = () => {
+  const router = useRouter();
+  const phoneLess = useMediaQuery(lightTheme.breakpoints.down('table_375'));
+  const isMobile = useMediaQuery(lightTheme.breakpoints.between('table_375', 'table_834'));
+  const isLight = useThemeContext().themeMode === 'light';
+  const handleOnclick = () => {
+    router.push('/');
+  };
+  return (
+    <Wrapper>
+      <ImageContainer>
+        <Image
+          src={isLight
+            ? (
+                (phoneLess || isMobile) ? BackgroundMobile404 : Background404)
+            : (
+                (phoneLess || isMobile) ? BackgroundMobile404Dark : Background404Dark
+              )}
+          objectFit="fill"
+          alt="404"
+          layout='fill'
+        />
+        <ContainerData>
+          <LogoContainer>
+            <Image src={isLight ? Logo404 : Logo404Dark}
+              layout='fill'
+              objectFit='contain'
+              objectPosition='center'
+            />
+          </LogoContainer>
+          <ContainerText>
+            <TextUps isLight={isLight}>Oops!</TextUps>
+            <TextDescription >The Page you requested couldn't be found</TextDescription>
+          </ContainerText>
+          <ContainerButton> <CustomButton widthText='100%' label='Go Back to Homepage' style={{
+            display: 'flex',
+            flexDirection: 'row',
+            justifyContent: 'center',
+            alignItems: 'center',
+            gap: '8px',
+            width: '250px',
+            height: '48px',
+            background: isLight ? '#E7FCFA' : 'transparent',
+            border: '1px solid #1AAB9B',
+            borderRadius: '22px',
+            // eslint-disable-next-line @typescript-eslint/no-empty-function
+          }} onClick={handleOnclick} styleText={{
+            color: '#1AAB9B',
+            fontSize: '16px',
+            lineHeight: '24px'
+          }} />
+          </ContainerButton>
+        </ContainerData>
+      </ImageContainer>
+    </Wrapper >);
+};
+
+const Wrapper = styled.div({
+  width: '100%',
+  height: '100%',
+  marginTop: '132px',
+  marginBottom: '128px',
+  [lightTheme.breakpoints.between('table_834', 'desktop_1440')]: {
```

Comment:
I just noted that you're already using `marginTop: '132px'` as a default value, so this media query is no longer needed